src/main/java/duke/utils/Parser.java

```diff
@@ -0,0 +1,195 @@
+package duke.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.regex.Pattern;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteAllCommand;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.ShowAllTasksOnSameDateCommand;
+import duke.command.UnmarkCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+/**
+ * The type Parser.
+ */
+public class Parser {
+    /**
+     * Instantiates a new Parser.
+     */
+    public Parser() {}
+
+    /**
+     * Parse input to Command.
+     *
+     * @param input the user input
+     * @return the specific Cortana command
+     * @throws CortanaException the cortana exception
+     */
+    public static Command parse(String input) throws CortanaException {
```

Comment:
Maybe you could shorten this method? Seems like a [long method](https://nus-cs2103-ay2122s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)